src/sentry/integrations/discord/client.py

```diff
@@ -102,6 +109,51 @@ def get_channel(self, channel_id: str) -> object | None:
             CHANNEL_URL.format(channel_id=channel_id), headers=self.prepare_auth_header()
         )
 
+    def track_response_data(
+        self,
+        code: str | int,
+        span: Span | None = None,
+        error: Exception | None = None,
+        resp: Response | None = None,
+        extra: Optional[Mapping[str, str]] = None,
+    ) -> None:
+        # if no span was passed, create a dummy to which to add data to avoid having to wrap every
+        # span call in `if span`
+        span = span or Span()
+
+        if code == status.HTTP_403_FORBIDDEN:
```

Comment:
As I mentioned offline, we might need to extend this function in the future. Btw, do you also want to exclude other status code we exclude in current SLO / monitoring?